models/public/ssd_resnet50_v1_fpn_coco/ssd_resnet50_v1_fpn_coco.md

```diff
@@ -0,0 +1,75 @@
+# ssd_resnet50_v1_fpn_coco
+
+## Use Case and High-Level Description
+
+The `ssd_resnet50_v1_fpn_coco` model is a SSD FPN object detection architecture based on Resnet50. The model has been trained from the Common Objects in Context (COCO) image dataset.
+For details see the [repository](https://github.com/tensorflow/models/blob/master/research/object_detection) and [paper](https://arxiv.org/abs/1708.02002).
+
+The model input is a blob that consists of a single image of 1x3x640x640 in RGB order.
+
+The model output is a typical vector containing the tracked object data, as previously described. Note that the "class_id" data is now significant and should be used to determine the classification for any detected object.
```

Comment:
not understand, where you got this phrase from and how is it connected with model?

Reply:
Fixed